.ci.yaml

```diff
@@ -3718,6 +3728,16 @@ targets:
         ["devicelab", "ios", "mac"]
       task_name: integration_ui_ios_driver
 
+  - name: Mac_ios integration_ui_ios_driver_xcode_debug
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "ios", "mac"]
+      task_name: integration_ui_ios_driver_xcode_debug
+    bringup: true
```

Comment:
ask to learn: what does bringup true mean?

Reply:
This means that *if* the test fails on post-submit, it will not block the tree. It is part of flutter CI policy when adding a new test target that it must have `bringup: true`, because frequently new tests will be misconfigured in some difficult to spot way.

Reply:
Hmmm, why the name "bringup". it sounds a bit odd to me.